mw-webapp/src/component/sidebar/SidebarTrigger/SidebarTrigger.tsx

```diff
@@ -0,0 +1,13 @@
+import {PropsWithChildren} from "react";
+import {Trigger as DialogTrigger} from "@radix-ui/react-dialog";
```

Comment:
@radix-ui/react-dialog does not added to dependencies in the package.json and it leads to an error even if I run "pnpm i" command